src/adc.py

```diff
@@ -0,0 +1,91 @@
+'''
+
+Python script to read analog voltage values from ADS1115 16bit ADC with PGA
+Author: Carson Berry 
+Date: February 2nd, 2021
+
+
+usage: 
+
+    python3
+    import src.adc as adc
+    sensors = adc.adc_sensors()
+    sensors.read_leak()
+    sensors.read_pH()
+
+
+
+Reference provided at: https://learn.adafruit.com/adafruit-4-channel-adc-breakouts/python-circuitpython
+
+'''
+
+import board
+import busio
+import adafruit_ads1x15.ads1115 as ADS
+from adafruit_ads1x15.analog_in import AnalogIn
+
+def init_i2c():
+   #define i2c object
+    i2c = busio.I2C(board.SCL, board.SDA)
+    #create object
+    ads = ADS.ADS1115(i2c) 
+    return ads
+
+class adc_sensors(object):
+    def __init__(self):
+        self.ads=init_i2c();
+        self.leak_sensor = init_leak(self.ads)
+        self.pH_sensor = init_pH(self.ads)
+       
+    def read_leak(self):
+        return self.leak_sensor.voltage
+
+    def read_pH(self):
+        return calc_pH(self.pH_sensor.voltage)
```

Comment:
Instead of having these functions being nested, try including the `calc_pH(...)` function in this class. This just improves encapsulation and clarity if you are going for an object oriented design. Same thing goes with the `init...` functions. All of that could be encapsulated in the `adc_sensors` class.